src/main/java/edu/kis/vh/nursery/FIFORhymer.java

```diff
@@ -1,21 +1,21 @@
 package edu.kis.vh.nursery;
 
-public class FIFORhymer extends defaultCountingOutRhymer {
-
-	public defaultCountingOutRhymer temp = new defaultCountingOutRhymer();
-	
-	@Override
-	public int countOut() {
-		while (!callCheck())
-			
-		temp.countIn(super.countOut());
-		
-		int ret = temp.countOut();
-		
-		while (!temp.callCheck())
-			
-		countIn(temp.countOut());
-		
-		return ret;
-	}
+public class FIFORhymer extends DefaultCountingOutRhymer {
+
+    public DefaultCountingOutRhymer temp = new DefaultCountingOutRhymer();
+
+    @Override
+    public int countOut() {
+        while (!callCheck())
+
+            temp.countIn(super.countOut());
+
+        int ret = temp.countOut();
+
+        while (!temp.callCheck())
+
+            countIn(temp.countOut());
+
+        return ret;
```

Comment:
Original project formatting convention used tabs (should not be changed to spaces). It makes tracking actual changes more difficult.